code/datums/jobs/job/terragov.dm

```diff
@@ -4,6 +4,8 @@
 /datum/job/terragov/radio_help_message(mob/M)
 	. = ..()
 	if(istype(SSticker.mode, /datum/game_mode/combat_patrol))
+		if(SSticker.mode.flags_round_type & MODE_SENSOR)
+			to_chat(M, span_highdanger("Your platoon has orders to attack this ao's sensor towers and reactivate them in order to alert other TerraGov forces in the sector about the invasion. High Command considers the successful reactivation of the sensor towers a major victory"))
```

Comment:
```suggestion
			to_chat(M, span_highdanger("Your platoon has orders to attack this sensor towers in the AO and reactivate them in order to alert other TerraGov forces in the sector about the invasion. High Command considers the successful reactivation of the sensor towers a major victory."))
```

Reply:
Did you just suggest a change to ADD a typo?

Why...?